domain/schema/model/sql/0015-charm.sql

```diff
@@ -214,10 +215,12 @@ CREATE TABLE charm_relation (
     REFERENCES charm_relation_role (id),
     CONSTRAINT fk_charm_relation_scope
     FOREIGN KEY (scope_id)
-    REFERENCES charm_relation_scope (id),
-    PRIMARY KEY (charm_uuid, kind_id, "key")
+    REFERENCES charm_relation_scope (id)
 );
 
+CREATE UNIQUE INDEX idx_charm_relation
```

Comment:
This index is perfect, I'd just name it `idx_charm_relation_charm_key` to include the "key" into its name.

Reply:
Done